packages/type/tsconfig.json

```
@@ -1,10 +1,9 @@
 {
 	"extends": "../ts-config/src/tsconfig.json",
 	"compilerOptions": {
-		"target": "ES2020",
-		"module": "ESNext",
 		"outDir": "build",
-		"rootDir": "src"
+		"rootDir": "src",
+		"importHelpers": true
```

```suggestion
		"rootDir": "src"
```

`importHelpers` is enabled by default

---

yeah, was fighting a bug at one point and tried it